sql/step_04_all_following_steps.sql

```diff
@@ -100,4 +100,5 @@ source step_a103_fix_MedicalDsrStockMovementTypeBrowser.sql;
 source step_a104_add_inventory_in_pharmacy.sql;
 source step_a105_medicaldsr_soft_deletion.sql;
 source step_a106_medicaldsrstock_control.sql;
-source step_a107_add_permission_admin_access.sql;
\ No newline at end of file
+source step_a107_add_permission_admin_access.sql;
+source step_a108_alter_table_dicom_type_and_dicom_data.sql;
```

Comment:
let's call it this way

```suggestion
source step_a108_audit_dicom_type_and_dicom_data.sql;
```